server/config/conditions.ts

```diff
@@ -624,6 +626,8 @@ export default {
             label: 'Select a review period',
             name: 'reviewPeriod',
             case: 'lower',
+            beforeVowel: 'an ',
```

Comment:
The value of `beforeVowel` and `beforeConsonant` will always be `a` and `and` respectively right? Instead of writing a value in each place its used, maybe just a flag like this?

`handleIndefiniteArticle: true`